addons/stock/models/stock_quant.py

```diff
@@ -127,6 +132,50 @@ def _compute_inventory_date(self):
         for quant in quants:
             quant.inventory_date = date_by_location[quant.location_id]
 
+    def _compute_last_count_date(self):
+        """ We look at the stock move lines associated with every quant to get the last count date.
+        """
+        groupby_vars = ['sml.product_id', 'sml.lot_id', 'sml.company_id', 'sml.location_id', 'sml.location_dest_id', 'sml.package_id', 'sml.result_package_id']
+        where_clause = f"""
+            move.is_inventory is true AND
+            sml.state = 'done' AND
+            sml.product_id in ({', '.join([str(x) for x in self.product_id.ids])}) AND
```

Comment:
never ever do that. Always use %s and pass the param through execute to avoid any sql injection

Reply:
even though there is no real exploit here but it's a good habit to take.